tools/install_bazel.sh

```diff
@@ -6,9 +6,18 @@ echo "Checking if Bazel ${BAZEL_VERSION} needs to be installed..."
 if [[ $(command -v bazel) && "$(bazel version | grep 'label' | cut -d ' ' -f 3)" =~ ${BAZEL_VERSION} ]]; then
   echo "Bazel ${BAZEL_VERSION} already installed, skipping fetch."
 else
-  wget -nv https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
-  chmod +x bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
-  sudo -E ./bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
-  sudo -E mv /usr/local/bin/bazel /usr/bin
-  rm bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
+  if [ "$(uname -m)" = "aarch64" ]; then
+     echo "Install Bazel on arm64: ......"
+     sudo apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y curl unzip zip build-essential openjdk-11-jdk
```

Comment:
You are still missing `sudo` in the second command

```suggestion
     sudo apt-get update && DEBIAN_FRONTEND=noninteractive sudo -E apt-get install -y curl unzip zip build-essential openjdk-11-jdk
```